modules/controlpanel.cpp

```diff
@@ -1130,7 +1132,7 @@ class CAdminMod : public CModule {
         CString sPass = sLine.Token(5);
 
         if (sServer.empty()) {
-            PutModule("Usage: DelServer <username> <network> <server>");
+            PutModule("Usage: DelServer <username> <network> <server> [port] [password]");
```

Comment:
That sounds like a flaw in the AddServer API for taking a uInt rather than a CString, I looked at `DelServer` (Which is newer code) and seemed to be more concrete, therefore it made more sense for me to refactor the old to use the new. You raise a valid concern though.